packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -88,11 +88,26 @@ interface ListViewProps<T> extends CollectionBase<T>, DOMProps, AriaLabelingProp
    * @default 'regular'
    */
   density?: 'compact' | 'regular' | 'spacious',
+  /** Whether the ListView should be displayed with a quiet style. */
   isQuiet?: boolean,
+  /** The current loading state of the ListView. Determines whether or not the progress circle should be shown. */
   loadingState?: LoadingState,
+  /** Sets what the ListView should render when there is no content to display. */
   renderEmptyState?: () => JSX.Element,
+  /**
+   * The duration of animated layout changes, in milliseconds. Used by the Virtualizer.
+   * @default 0
+   */
   transitionDuration?: number,
```

Comment:
As an aside, I don't think this should be in the ListView api, not sure why it is here in the first place. Probably should  get removed and match TableView, will do in a separate PR

Reply:
Definitely agree